### ISSUE TITLE
HotKey labels not in Outline

### ISSUE THREAD

**Issue by BartmanEH** [mark-wiemer/ahkpp#87]
## Description
HotKey labels arguably are just another sort of label:
![image](https://user-images.githubusercontent.com/6135879/116725108-845d2580-a9af-11eb-9630-18ab8d670334.png)
However, HotKey labels do not appear in the Outline:
![image](https://user-images.githubusercontent.com/6135879/116725213-a5257b00-a9af-11eb-8799-51af9bbbaefa.png)

**Comment by JoKone** [comment]
This is a very useful feature to have. The NekoHelp extension demonstrates this. If it is not too complicated, please include this here as well.

**Comment by Giessen** [comment]
Hotkey labels still not apearing in outline.

**Comment by mark-wiemer** [comment]
Correct. I'll update the issue when this is fixed, but it may be a while. Remember I'm just one guy!!
